### DISCUSSION TITLE
Configure isolation level on connector-x and the number of python threads

### DISCUSSION THREAD

**Discussion by ritalinux** [sfu-db/connector-x#551] [Q&A]
Hey guys! Is there any way to configure MySQL isolation level on connector-x with Polars?

I want to start a lot of queries in db, until now I use sqlalchemy-pymysql with isolation level READ UNCOMMITTED config because I don't want to block any table in production db.

Is it possible to configure this in polars.read_database_uri()?

Other thing is, connector-x make use of ThreadPoolExecutor under the hood? Or use something that I can configure the number of the threads to makes the queries?

**Comment by Joelciomatias** [comment]
> Hey guys! Is there any way to configure MySQL isolation level on connector-x with Polars?
> 
> I want to start a lot of queries in db, until now I use sqlalchemy-pymysql with isolation level READ UNCOMMITTED config because I don't want to block any table in production db.
> 
> Is it possible to configure this in polars.read_database_uri()?
> 
> Other thing is, connector-x make use of ThreadPoolExecutor under the hood? Or use something that I can configure the number of the threads to makes the queries?

There are any solution for this issue, as you said, we do not want to block any table in production db, please help community.
